src/MongoDB/Connection.php

```diff
@@ -38,7 +37,7 @@ class Connection extends BaseConnection
     public function __construct(Client $client, $database, $collectionName = '', array $config = [])
     {
         $this->client = $client;
-        $this->collectionName = $collectionName;
+        $this->collection_name = $collectionName;
```

Comment:
are properties supposed to be snake cased according to the coding standards?

Reply:
Yes.  ParTech coding standards should apply here.